.travis.yml

```diff
@@ -52,3 +53,5 @@ script:
   - cd docs && make html && cd ..
   # Make sure the library installs.
   - python setup.py install
+  # Run benchmark
+  - python bin/benchmark_pydy_code_gen.py 4 100
```

Comment:
not really

On Sat, Oct 17, 2015 at 10:21 PM, Jason K. Moore <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/pydy/pydy/pull/277#discussion_r42313809>:
>
> > @@ -52,3 +53,5 @@ script:
> >    - cd docs && make html && cd ..
> >    # Make sure the library installs.
> >    - python setup.py install
> > +  # Run benchmark
> > +  - python bin/benchmark_pydy_code_gen.py 4 100
>
> You could make this: 1 10 so it runs faster. Any reason we need this to
> run with 4 links and 100 time steps?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydy/pydy/pull/277/files#r42313809>.
>